sky/compositor/BUILD.gn

```diff
@@ -24,6 +24,8 @@ source_set("compositor") {
     "picture_layer.h",
     "transform_layer.cc",
     "transform_layer.h",
+    "utilities/picture_rasterizer.cc",
+    "utilities/picture_rasterizer.h",
```

Comment:
I would just put this in the root directory for now.  We can refactor layer.  Directories called `utilities` are an attractive nusiance